documentation/swagger-authoring-manual-guidelines.md

```diff
@@ -0,0 +1,52 @@
+# Manual Guidelines for Swagger Authoring #
+
+This document lists the set of manual guidelines that need to be followed while authoring swagger specs. 
+
+[Swagger validation tools](https://github.com/Azure/adx-documentation-pr/blob/master/tools/tools.md) validate a large set of rules. These are listed in [Automated rules document](swagger-authoring-automated-guidelines.md).
+
+Manual guidelines are the rules that could not be automated and would have to be verified manually by swagger authors before sending a pull request. 
+
+It is a requirement to conform to all manual and automated rules with severity "Error" before sending a pull request for review.
+
+## Error vs Warning
+- Rules with severity "Error" have to be addressed for the swagger to be approved by the reviewers. If there is a strong reason for why the rule cannot be addressed in a swagger spec, escalation process has to be followed to get sign off from management.
+
+- Rules with severity "Warning" are strong recommendations made by Azure developer experience team for better SDK/Documentation experience. It is NOT a requirement to address them.
+
+## Manual Rules
+
+| Rule | Severity | Category |
+| --- | --- | --- |
+| Resource Model name MUST be same as singular form of the resource type. Like for virtualMachines resource type it should be VirtualMachine | Error |  |
```

Comment:
There are still too many manual rules in the list below, I think we should look for a way to simplify or reformat the below, and of course, if there are more to automate we would. I think it may be helpful to group the ones that will be "automated soon" too.